Sources/Basic/PathShims.swift

```diff
@@ -21,64 +21,10 @@ import SPMLibc
 import POSIX
```

Comment:
Does this need to be removed as well?

Reply:
Probably, but, can't as of yet, I'm just mechanically removing the items so that we can split this up and merge it piecemeal.  It's easier to review this an API at a time rather than everything at once.